README.md

```diff
@@ -241,17 +267,112 @@ Refer to the following README sections on push setup:
 
 ### Collecting Push Tokens
 
-Push tokens must be collected in the native layer. Follow the platform-specific instructions below:
+Push tokens must be collected both in your app's react native code or in the native code. Below sections discuss both approaches, and
+you are free to pick one that best suits your app's architecture. Note that doing this in one location is sufficient.
+
+#### React Native Token Collection
+
+In order to collect the APNs push token in your React Native code you need to,
+
+1.  Import libraries such as [`@react-native-firebase/messaging`](https://www.npmjs.com/package/@react-native-firebase/messaging) to your react native project. The below instructions are specific for `@react-native-firebase/messaging` library.
+2.  Import Firebase iOS SDK to your iOS project. Setup instructions can be found [here](https://firebase.google.com/docs/ios/setup).
+3.  In order for the Klaviyo SDK to handle push opens via the notification center delegate methods, we need to disable
```

Comment:
Do we want information about push opens in the token collection section?

Reply:
I wonder the same... spitballing here but would it make sense to just have a section  `Integrate with React Native Firebase` instead maybe

Reply:
> Do we want information about push opens in the token collection section?

Yeah, I was not fully convinced as well but did seems related to method swizzling topic which is discussed here. May be this can be reworded to just say that method swizzling should be disabled in order for you notification center delegates in AppDelegate to be called.

Reply:
Oh yeah that should sound better

Reply:
@evan-masseau  I was trying to keep the instructions pretty loose and just suggest firebase as an option w/o recommending it.

Reply:
Yeah that's a good point better to keep it generic than have a specific firebase section